### ISSUE TITLE
Swap disks are not formatted correctly on Ceph

### ISSUE THREAD

**Issue by OpenNebulaSupport** [OpenNebula/one#6090]
**Description**
Volatile swap disks are not swap formatted on a Ceph datastore.

**To Reproduce**
1. Create a VM template with a swap disk type.
2. Instantiate the VM on a Ceph datastore.
3. The swap disk is not formatted correctly:

    ```
    localhost:~# lsblk
    NAME   MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
    sr0     11:0    1  364K  0 rom  
    vda    253:0    0  256M  0 disk 
    └─vda1 253:1    0  255M  0 part 
    vdb    253:16   0    1G  0 disk 
    ```

**Expected behavior**
The swap disk should be correctly formatted as swap device:
```
localhost:~# fdisk -l
Disk /dev/vda: 256 MiB, 268435456 bytes, 524288 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes
Disklabel type: dos
Disk identifier: 0x99c507c2

Device     Boot Start    End Sectors  Size Id Type
/dev/vda1  *     2048 524287  522240  255M 83 Linux

Disk /dev/vdb: 1 GiB, 1073741824 bytes, 2097152 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes

localhost:~# blkid -t TYPE="swap" -o device
/dev/vdb
```
**Details**
 - Affected Component: Ceph driver
 - Hypervisor: KVM
 - Version: 6.4.2

## Progress Status
- [ ] Code committed
- [ ] Testing - QA
- [ ] Documentation (Release notes - resolved issues, compatibility, known issues)

**Comment by lgrawet** [comment]
Hello,

It appears the following path
    /VM/TEMPLATE/DISK[DISK_ID=$DISK_ID]/TYPE
is not handled by the mkimage script. The script is lacking a conditional which would call the mkswap script when TYPE=swap.

**Comment by lgrawet** [comment]
Hello,

Here is a patch.
[ceph-swap.patch.txt](https://github.com/OpenNebula/one/files/10518162/ceph-swap.patch.txt)

Best regards,

Laurent

**Comment by rsmontero** [comment]
Solved as per 1eb8e75c46b522fc888ec2fab9dfe636171f0990

@lgrawet THANKS!!
